<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileTest</class>
 <widget class="QDialog" name="FileTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Тестирование ОПУ</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Файл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leFileName">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bOpenFile">
       <property name="text">
        <string>Открыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Число тестов:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lNumberTest">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Успешных тестов:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lSuccess">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="bStart">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Запустить тестирование</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bStop">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Остановить тестирование</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="tbInfo"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
